src/kubernetes_cluster/spec/cluster.rs

```diff
@@ -282,12 +321,12 @@ pub open spec fn busy_kubernetes_api_rejects_request<K: ResourceView, R: Reconci
 /// This action disallows the kubernetes api to be busy from this point.
 /// This is used to constraint the crash behavior for liveness proof:
```

Comment:
I should have caught this earlier: "This is used to constraint the crash behavior for liveness proof" -> "This is used to constraint the transient error of Kubernetes APIs for liveness proof"